public/blog/testrpc-is-now-ganache-cli.md

```diff
@@ -0,0 +1,25 @@
+# TestRPC is now Ganache CLI
+
+Have you been looking for the TestRPC recently and haven't been able to find anything about it?
```

Comment:
Should probably change to just `find it`. There's a ton of other websites that give information _about_ it, but the Github repo and name has changed. 

(nitpicky/artistic difference?)